.github/workflows/versioning.yml

```
@@ -0,0 +1,42 @@
+  push:
+    branches:
+      - development
+    types:
+      - merged
+
+jobs:
+  bump_version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)